micro_sam/evaluation/util.py

```diff
@@ -0,0 +1,43 @@
+import os
+from typing import Union, Optional, List
+
+from . import instance_segmentation, inference
+from ..instance_segmentation import AutomaticMaskGenerator
+
+
+def run_amg(
```

Comment:
This function seems to be identical to https://github.com/computational-cell-analytics/micro-sam/blob/7d6010fb95533d5293769b32131a5be8d674e519/micro_sam/evaluation/inference.py#L528

Can we remove it (including this `util.py`) and use the `run_amg` from there instead?

Reply:
Yes, forgot to remove that. Fixed. Thanks for spotting!